utils/appliance/__init__.py

```diff
@@ -2215,19 +2204,13 @@ def push(self, obj):
 
         logger.info("Pushed appliance {} on stack (was {} before) ".format(
             obj.address, getattr(was_before, 'address', 'empty')))
-        if obj.browser_steal:
```

Comment:
I _think_ we may need this, what mechanism is in place to restart the browser on the context switch? One of them takes the URL key which I don't think we are properly utilising yet, that comes in my configuration PR I believe